### PR TITLE
Pep8

### PR DESCRIPTION
**Pull request by BRJMM**

### CODE REVIEW

File: eyes-servo-tes.py
Comment (degv364) [inline eyes-servo-tes.py]: Functions are all lower case

File: eyes-servo-tes.py
Comment (degv364) [inline eyes-servo-tes.py]: Lower case plase

File: eyes-servo-tes.py
Comment (degv364) [inline eyes-servo-tes.py]: lower case please

File: eyes-servo-tes.py
Comment (degv364) [inline eyes-servo-tes.py]: lower case please

File: eyes-servo-tes.py
Comment (degv364) [inline eyes-servo-tes.py]: lower case please

File: eyes-servo-tes.py
Comment (degv364) [inline eyes-servo-tes.py]: lowercase please

File: eyes-servo-tes.py
Comment (degv364) [inline eyes-servo-tes.py]: lowercase please

**Review by degv364 — CHANGES_REQUESTED** [review]
According to PEP8 function names should be in lowercase.

**Comment by BRJMM** [comment]
changes made

**Review by degv364 — APPROVED** [review]

**Comment by degv364** [comment]
Looks good to me. But lets wait for another reviewer to approve.

**Comment by BRJMM** [comment]
I will wait the another approve to continue

**Comment by jprltsnz** [comment]
Changed merged, thanks!

**Pull request merged**
